Formula/uriparser.rb

```diff
@@ -1,8 +1,16 @@
 class Uriparser < Formula
   desc "URI parsing library (strictly RFC 3986 compliant)"
   homepage "https://uriparser.github.io/"
-  url "https://github.com/uriparser/uriparser/releases/download/uriparser-0.9.1/uriparser-0.9.1.tar.bz2"
-  sha256 "75248f3de3b7b13c8c9735ff7b86ebe72cbb8ad043291517d7d53488e0893abe"
+  head "https://github.com/uriparser/uriparser.git"
```

Comment:
You removed the build dependencies for HEAD. You should also remove the `head` statement here, or add back the build dependencies conditioned on `build.head?`.